srcpkgs/qtile/INSTALL.msg

```diff
@@ -1,9 +1,35 @@
-!!! Config breakage !!!
-    - lazy.qtile.display_kb() no longer receives any arguments. If you passed
-      it any arguments (which were ignored previously), remove them.
-    - If you have a custom startup Python script that you use instead of
-      "qtile start" and run init_log manually, the signature has changed.
-      Please check the source for the updated arguments.
-    - "KeyChord"'s signature has changed. "mode" is now a boolean to indicate
-       whether the mode should persist. The "name" parameter should be used
-       to name the chord (e.g. for the ``Chord`` widget).
+!!! Config breakage/changes !!!
+    - The `cmd_` prefix has been dropped from all commands (this means command names
+      are common when accessed via the command interface or internal python objects).
+    - Custom widgets should now expose command methods with the `@expose_command`
+      decorator (available via `from libqtile.command.base import expose_command`).
+    - Some commands have been renamed (in addition to dropping the 'cmd_' prefix):
+        `hints` -> `get_hints`
+        `groups` -> `get_groups`
+        `screens` -> `get_screens`
+    - Layouts need to rename some methods:
+        - `add` to `add_client`
+        - `cmd_next` to `next`
+        - `cmd_previous` to `previous`
+    - Layouts or widgets that redefine the `commands` property
+      need to update the signature:
+        `@expose_command()`
+        `def commands(self) -> list[str]:`
+    - `Window.getsize` has been renamed `Window.get_size`
+      (i.e. merged with the get_size command).
+    - `Window.getposition` has been renamed `Window.get_position`
+      (i.e. merged with the get_position command).
+    - The `StockTicker` widget `function` option is being deprecated: rename it to `func`.
+    - The formatting of `NetWidget` has changed, if you use the `format` parameter
+      in your config include `up_suffix`, `total_suffix` and `down_suffix` to display
+      the respective units.
+    - The `Notify` widget now has separate `default_timeout` properties for
+      differenct urgency levels. Previously, `default_timeout` was `None`
+      which meant that there was no timeout for all notifications (unless this
+      had been set by the client sending the notification). Now, `default_timeout`
+      is for normal urgency notifications and this has been set to a default of
+      10 seconds. `default_timeout_urgent`, for critical notifications,
+      has a timeout of `None`.
+    - The `PulseVolume` widget now depends on a third party library, `pulsectl-asyncio`,
+      to interact with the pulse audio server. Users will now see an `ImportError`
+      until they install that library.
```

Comment:
This project get breakage with every release. Does it help to have this message. IMO, no.

Reply:
My config did not load without doing changes documented in the list.
But for me it doesn't matter if it's included or not since I read them anyway to update the template.

Reply:
I removed the file.